index.html

```diff
@@ -2378,6 +2405,17 @@ <h2>
                 in Australia.
               </td>
             </tr>
+            <tr>
+              <td>
+                <dfn>[[\regionCode]]</dfn>
+              </td>
+              <td>
+                A <a>region</a> represented as a [[!ISO3166-2]] subdivision
+                code or the empty string, stored in its canonical uppercase
+                form. For example, "<code>PT-11</code>" represents the Lisbon
```

Comment:
"or the empty string" at the end of the sentence, please. There is no canonical uppercase form for the empty string.